salt/client/ssh/client.py

```diff
@@ -196,3 +197,41 @@ def cmd_async(self, low, timeout=None):
         '''
         # TODO Not implemented
         raise SaltClientError
+
+    def cmd_subset(
```

Comment:
This is just to mirror the setup of LocalClient().cmd_subset().

This is used from the _exec function in salt.modules.saltutil  Similarly to how cmd_batch is used.